src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -762,6 +762,12 @@ public static class MsalError
         /// </summary>
         public const string BrokerApplicationRequired = "broker_application_required";
 
+        /// <summary>
+        /// <para>What happens?</para>You are trying to authenticate with the broker but MSAL is unable to read the response from the broker.
+        /// <para>Mitigation</para>The currently installed broker may not support MSAL.Xamarin, you need to ensure that you have installed either Intune Company Portal (5.0.4689.0 or greater) or Microsoft Authenticator (6.2001.0140 or greater). see https://aka.ms/Brokered-Authentication-for-Android"
+        /// </summary>
+        public const string FailedToGetBrokerResponse = "failed_to_get_broker_response";
```

Comment:
nit: since this is just for android, maybe include "android" in the response string "failed_to_get_android_broker_response"